test/Microsoft.AspNetCore.Mvc.Razor.Test/Internal/FileProviderRazorProjectTest.cs

```diff
@@ -14,14 +16,14 @@ public class FileProviderRazorProjectTest
         public void EnumerateFiles_ReturnsEmptySequenceIfNoCshtmlFilesArePresent()
         {
             // Arrange
-            var fileProvider = new TestFileProvider();
+            var fileProvider = new TestFileProvider("BasePath");
```

Comment:
Can we add a couple of tests where the path does not belong to `ContentRootPath`?